.gitignore

```diff
@@ -15,3 +15,4 @@ coverage.xml
 fdjangodog_lint.txt
 testresults_*.xml
 
+.vscode/
```

Comment:
You should be able to hide this in your local .gitignore rather than (presumably) needing to add it to every project you work on. I'd rather keep the project gitignore file for stuff produced by the project.